backend/migrations/001_initial.sql

```diff
@@ -0,0 +1,146 @@
+-- +goose Up
+
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS Organisations (
```

Comment:
Table and column names must be in lower case with an underscore. It is the convention used by GORM for mapping models to tables

For example:
github_id
avatar_url

Reply:
Oh, i didn't know that, am updating !